sdks/python/apache_beam/io/iobase_test.py

```diff
@@ -136,14 +139,20 @@ def test_current_restriction_after_split(self):
                      self.sdf_restriction_tracker.current_restriction())
 
   def test_try_split_at_remainder(self):
-    fraction_of_remainder = 0.5
-    expected_primary = (0, 3)
-    expected_residual = (3, 4)
-    self.sdf_restriction_tracker.try_claim(1)
+    fraction_of_remainder = 0.4
```

Comment:
Did the previous values not work? Or did it not test what we wanted to test (in which case perhaps we should add a new test, not mutate the existing one)?

Reply:
0.5 is a tricky value since 0.5 == 1 - 0.5, which makes it not a good number to test how to treat `fraction_of_remainder`.

Reply:
Ack. Sounds good.